bokeh/embed.py

```diff
@@ -197,19 +197,37 @@ def file_html(plot_object, resources, title, template=FILE, template_variables=N
     if not isinstance(plot_object, (PlotObject, Document)):
         raise ValueError('plot_object must be a single PlotObject')
 
-    plot_resources = RESOURCES.render(
-        js_raw = resources.js_raw,
-        css_raw = resources.css_raw,
-        js_files = resources.js_files,
-        css_files = resources.css_files,
-    )
+    bokeh_js = ""
+    bokeh_css = ""
+
+    try:
+        bokeh_js = JS_RESOURCES.render(
+            js_raw=resources.js_raw,
+            js_files=resources.js_files,
+        )
+    except AttributeError:
```

Comment:
This way any misspelled name under this try-except block will go unnoticed. Ask explicitly with `hasattr()` if attributes exist.